components/infobox/wikis/starcraft2/infobox_person_user_custom.lua

```diff
@@ -7,48 +7,19 @@
 --
 
 local Class = require('Module:Class')
-local CleanRace = require('Module:CleanRace')
 local Lua = require('Module:Lua')
-local RaceIcon = require('Module:RaceIcon').getBigIcon
 local String = require('Module:StringUtils')
 local Variables = require('Module:Variables')
 
+local PersonSc2 = require('Module:Infobox/Person/Custom/Shared/dev')
```

Comment:
/dev

Reply:
ah forgot to change it back from teesting (atm /dev infoboxes are broken somewhere down the line so tested with invoking the /dev Custom(s) plus the /dev of the shared stuff and the rest from live)